Sources/Extensions/UIKit/UIAlertControllerExtensions.swift

```diff
@@ -17,20 +17,74 @@ import AudioToolbox
 // MARK: - Methodss
 public extension UIAlertController {
 
-	/// SwifterSwift: Present alert view controller in the current view controller.
-	///
-	/// - Parameters:
-	///   - animated: set true to animate presentation of alert controller (default is true).
-	///   - vibrate: set true to vibrate the device while presenting the alert (default is false).
-	///   - completion: an optional completion handler to be called after presenting alert controller (default is nil).
-	public func show(animated: Bool = true, vibrate: Bool = false, completion: (() -> Void)? = nil) {
-		UIApplication.shared.keyWindow?.rootViewController?.present(self, animated: animated, completion: completion)
-		if vibrate {
-			#if canImport(AudioToolbox)
-			AudioServicesPlayAlertSound(kSystemSoundID_Vibrate)
-			#endif
-		}
-	}
+  // swiftlint:disable next cyclomatic_complexity
+  /// SwifterSwift: Present alert view controller.
+  ///
+  /// - Parameters:
+  ///   - viewController: a custom view controller to be presented (default is the rootViewController of application's key window).
+  ///   - anchor: specify the anchor for the popover’s arrow points to (default is nil).
```

Comment:
Perhaps you should mention the different value options. Honestly I think that this would be better as an `enum` that contains the different possible value types, but it's not essential to do.

Reply:
I agree with your suggestion, please make a decision and tell me if I want to do it.